ui/components/MesheryMeshInterface/PatternService/RJSF.js

```diff
@@ -18,26 +33,51 @@ function RJSFButton({ handler, text }) {
   );
 }
 
-function RJSF({ jsonSchema, onChange, hideSubmit, hideTitle, onSubmit, onDelete }) {
+const uiSchema = {
+  replicas: {
+    "ui:widget" : "range"
+  }
+};
+
+function RJSF({ jsonSchema, onChange, hideSubmit, hideTitle, onSubmit, onDelete, isMeshery }) {
   const [data, setData] = React.useState();
 
   React.useEffect(() => {
     onChange?.(data);
   }, [data]);
 
   return (
-    <Form
-      schema={hideTitle ? deleteTitleFromJSONSchema(jsonSchema) : jsonSchema}
-      idPrefix={jsonSchema?.title}
-      onChange={(e) => setData(e.formData)}
-      formData={data}
-      liveValidate
-      additionalMetaSchemas={[JS4]}
-      // noHtml5Validate
-    >
-      {hideSubmit ? true : <RJSFButton handler={onSubmit} text="Submit" />}
-      {hideSubmit ? true : <RJSFButton handler={onDelete} text="Delete" />}
-    </Form>
+    <>
+      {isMeshery ? (
```

Comment:
@Jashpatel1 `isMeshery` seems a lil bit confusing for the boolean. because we will always be using meshery in any case wdyt?

Reply:
What is being created here is an extension point. There are a number of extension points in Meshery, which can be considered an extensible platform.￼ incorporating use of the term "extension point" or "remote provider"￼ or "extension" would be good.

Reply:
@Jashpatel1 lets change this variable name to something more logical ?